src/classes/HiddenCommand.ts

```diff
@@ -62,48 +63,55 @@ export abstract class HiddenCommand {
    * @param epic 彩蛋回應
    * @returns 是否成功回應（必定為 `true`）
    */
-  protected epicResponse(message: Message, notEpic: (string | MessageCreateOptions)[], epic: (string | MessageCreateOptions)[]): true {
-    message.channel.send(
-      randomInt(1, 1000) <= 2 ?
-        randomElement(epic) :
-        randomElement(notEpic)
-    );
+  protected allTimeResponse(message: Message, notEpic: HiddenResponse[], epic: HiddenResponse[]): true {
+    const response = Math.random() < 0.002 ? randomElement(epic) : randomElement(notEpic);
```

Comment:
all time response 應該是 2%
```suggestion
const response = Math.random() < 0.02 ? randomElement(epic) : randomElement(notEpic);
```